env/GOOGLEPW.env

```
@@ -43,6 +43,17 @@ if [[ "${step}" = "fcst" ]] || [[ "${step}" = "efcs" ]]; then
     export APRUN_UFS="${launcher} -n ${ufs_ntasks}"
     unset nnodes ufs_ntasks
 
+elif [[ "${step}" = "prep_emissions" ]]; then
+
+    export APRUN="${APRUN}"
```

```suggestion
    export APRUN
```
should be sufficient, i.e. there is no need to assign the variable to itself to export it.

---

Agreed, and changed.